src/main/java/duke/MainWindow.java

```diff
@@ -0,0 +1,65 @@
+package duke;
+
+import duke.ui.Ui;
+import javafx.fxml.FXML;
+import javafx.scene.control.Button;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.image.Image;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.VBox;
+
+/**
+ * Controller for MainWindow. Provides the layout for the other controls.
+ */
+public class MainWindow extends AnchorPane {
+    @FXML
+    private ScrollPane scrollPane;
+    @FXML
+    private VBox dialogContainer;
+    @FXML
+    private TextField userInput;
+    @FXML
+    private Button sendButton;
+
+    private Duke duke;
+
+    private Image userImage = new Image(this.getClass().getResourceAsStream("/images/avatar.jpg"));
+    private Image dukeImage = new Image(this.getClass().getResourceAsStream("/images/avatar.jpg"));
+
+    /**
+     * Sets up the javafx ui widgets.
+     *
+     */
+
```

Comment:
The Javadoc comment should follow closely before the method! So I think it would be clearer if there where no newlines between the javadoc comment and its following method.